index.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+
+module.exports = class LambdaInvokeAsync {
+  constructor(options) {
+    options = options || {};
+    this._client = new AWS.Lambda();
+    this.logger = options.logger;
+  }
+
+  _formatClientContext(context) {
+    if (!context) {
+      return null;
+    }
+    return new Buffer(JSON.stringify(context)).toString('base64');
+  }
+
+  invokeAsync(functionName, payload, invocationType, clientContext) {
```

Comment:
Why no to follow the same pattern as in the other library (with callback)? I'm not a big fan of callbacks, but just to be consistent.